test/e2e/components/vl-icon.js

```diff
@@ -0,0 +1,28 @@
+const VlElement = require('./vl-element');
+
+class VlIcon extends VlElement {
```

Comment:
Nog e2e testen toevoegen. Ik denk aan:
1. als gebruiker wil ik het verschil kunnen zien tussen een klein en een gewoon icon
2. als gebruiker wil ik het verschil kunnen zien tussen een licht en een gewoon icon
3. als gebruiker wil ik het verschil kunnen zien tussen een groot en een gewoon icon
4. als gebruiker wil ik het verschil kunnen zien tussen een icoon dat 90 graden gedraaid is en een gewoon icon
5. als gebruiker wil ik het verschil kunnen zien tussen een icoon dat 180 graden gedraaid is en een gewoon icon
6. als gebruiker wil ik het verschil kunnen zien tussen een icoon waar gedefinieerd is dat het voor een tekst staat en een gewoon icoon
7. als gebruiker wil ik het verschil kunnen zien tussen een icoon waar gedefinieerd is dat het achter een tekst staat en een gewoon icoon

Reply:
Ik was hier nog niet aan begonnen gezien we met de button bezig waren en we een basic icon release moesten hebben met de nieuwe opzet. Testen volgen nog.